src/Video/ParticleSystemRenderer.cpp

```diff
@@ -18,12 +18,12 @@ ParticleSystemRenderer::ParticleSystemRenderer() {
     srand(time(NULL));
 
     // Temporary rotation matrix for future rotation of particles.
-    rotMat[0][0] = cos(1);
-    rotMat[0][1] = -sin(1);
+    rotMat[0][0] = static_cast<float>(cos(1));
```

Comment:
Maybe there are `sinf` and `cosf` functions for this and the upcoming three occurences.